### DISCUSSION TITLE
ctags

### DISCUSSION THREAD

**Discussion by rahil627** [focus-editor/focus#446] [Ideas]
i understand the editor wants to stay lean, and i'm afraid an lsp may be too heavy, but i wonder, are ctags light-weight enough..?

**Comment by focus-editor** [comment]
We're planning a test run of very basic lsp support after version 0.4.0.
If it proves to be a problem we will remove it.

**Reply by rahil627** [reply]
ooooh, wow.  i.. did not expect that..! amazing!! i love how many little design nuances there are.. 😄 

yeah, i partly ask because i've got a weird case of a game scripting lang that doesn't have an lsp, just ctags :cry: but lsp is def the way to go..! i guess i'll close this, as ctags is pretty much deprecated. 🫤
